docs/source/cmake/cmake_packages_without_sudo.rst

```diff
@@ -0,0 +1,95 @@
+
+
+Install a CMake package without sudo privileges
+===============================================
+
+To install a CMake package or library without sudo privileges, we need to define a directory to which we have access. For instance, 
+in :code:`~/`.
+
+
+Create a custom folder
+----------------------
+
+In this tutorial, we are going to create a custom folder :code:`~/opt` containing 
+the folders :code:`lib` and :code:`include`. This will be our directory to install all our CMake packages.
+
+
+.. code-block:: console
+
+    cd ~/
+    mkdir opt && cd opt
+    mkdir include
+    mkdir lib
+
+Then, we update the LD_LIBRARY_PATH in :code:`~/.bashrc`.    
+
+.. code-block:: console
+
+    echo "export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:~/opt/lib" >> ~/.bashrc
+    source ~/.bashrc
+
+
+.. image:: ../../images/cmake_without_sudo_steps.gif
+   :align: center    
+
+
+Install a CMake package
+-----------------------
+
+To install a CMake package, we set the :code:`CMAKE_INSTALL_PREFIX:PATH` flag with our custom folder (:code:`/home/USERNAME/opt`)
+
+
+.. code-block:: console
+
+    cmake -DCMAKE_INSTALL_PREFIX:PATH=~/opt .. 
+    make -j16
```

Comment:
@juanjqo Just in case might be better to leave `-j16` out or explain what the numbers mean otherwise a computer with fewer cores might decide to start a space program or catch on fire.